tests/10apidoc/30room-create.pl

```diff
@@ -374,7 +377,8 @@ sub room_alias_fixture
 
    return fixture(
       requires => [
-         room_alias_name_fixture( prefix => $args{prefix} ), $main::HOMESERVER_INFO[0],
+         room_alias_name_fixture( prefix => $args{prefix} ),
+         exists($args{remote}) ? $main::HOMESERVER_INFO[1] : $main::HOMESERVER_INFO[0],
```

Comment:
this will use `HOMESERVER_INFO[1]` even if `remote == 0`.

missing values in a map return `undef` which is falsy, so I think you can just do this to achieve what you want:

```suggestion
         $args{remote} ? $main::HOMESERVER_INFO[1] : $main::HOMESERVER_INFO[0],
```